api/server/models/user.js

```diff
@@ -0,0 +1,116 @@
+var GENERIC = require('../modules/generic');
+
+module.exports = function (User) {
+    User.register_login = function (action, action_type, social_id, platform, device_id, token, email_id, name, password, callback) {
```

Comment:
why does this have so many parameters?
it should be in json format instead. change this